pkgs/development/python-modules/rasterio/default.nix

```diff
@@ -0,0 +1,28 @@
+{ buildPythonPackage, lib, fetchFromGitHub,
+  cython,
+  numpy, affine, attrs, cligj, click-plugins, snuggs, gdal,
+  pytest, pytestcov, packaging, hypothesis, boto3
+}:
```

Comment:
Commas should be at the start, not end, of lines. (See the [manual](https://nixos.org/nixpkgs/manual/#sec-syntax).)

Reply:
@worldofpeace you marked this as resolved, but unless GitHub is being broken, it doesn't seem to have been fixed?